includes/Core/Authentication/Clients/Client_Factory.php

```diff
@@ -97,7 +104,6 @@ public static function create_client( array $args ) {
 		$client->setPrompt( 'consent' );
 		$client->setRedirectUri( $args['redirect_uri'] );
 		$client->setScopes( (array) $args['required_scopes'] );
-		$client->prepareScopes();
```

Comment:
This method didn't actually do anything meaningful as it has no side-effects, it only returns the list of requested scopes as a single string.